src/pages/homepage.ts

```diff
@@ -5,9 +5,15 @@ export type TestEvent = {
   foo: boolean
 }
 
+export const ViewChangeEvent = 'VIEW_CHANGE_EVENT'
```

Comment:
These are all events, no need to suffix it 
```suggestion
export const ViewChange = 'VIEW_CHANGE'
```